src/dataflow-types/src/logging.rs

```diff
@@ -23,6 +23,13 @@ pub struct LoggingConfig {
     pub log_logging: bool,
 }
 
+impl LoggingConfig {
+    /// Announce the identifiers the logging config will populate.
+    pub fn log_identifiers<'a>(&'a self) -> impl Iterator<Item = GlobalId> + 'a {
+        self.active_logs.values().cloned()
```

Comment:
`GlobalId` should be `Copy`, so it seems to me that `clone` shouldn't be needed here. I might be mistaken!

Reply:
I think `values()` returns references, so that is all that is going on (could also be `.map(|x| *x)`).

Reply:
Or `copied()`! Not that it matters.